README.md

```diff
@@ -16,6 +16,17 @@ The result of the sync is logged to a sync log available from the settings scree
 
 Only simple products are synced. Configurable products are not synced.
 
+## Requirements
+
+In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
+declared in the crontab via the following:
+
+php cron.php -mdefault 1
```

Comment:
what is the full format of the line in crontab as it should appear? can we give a command to modify the crontab. something like this

```
(crontab -l ; echo "0 * * * * php /path/to/magento/app/reverb/cron.php") | crontab -
```
I made up the path as I'm not sure where that would live.